content/docs/gto/get-started.md

```diff
@@ -0,0 +1,170 @@
+---
+description:
+  'Learn how you can use GTO to create Artifact Registry in Git repository'
+---
+
+# Get Started
+
+GTO helps you build an Artifact Registry on top of a Git repository. You can
+register relevant versions of your files or directories from any source (e.g. ML
+model releases) and assign them to different deployment environments (testing,
+shadow, production, etc.). Git-native mechanisms are used, so you can automate
+the delivery of your software project with CI/CD, and adopt a GitOps approach in
+general.
+
+This Get Started will walk you through basic GTO concepts and actions you would
+like to do in the Artifact Registry.
```

Comment:
The intro paragraph is already long so this utility sentence can go away I think. This way we get to actionable sections faster:

```suggestion
```